src/System.Management.Automation/engine/pipeline.cs

```diff
@@ -278,9 +277,12 @@ internal int Add(CommandProcessorBase commandProcessor)
 
         internal void AddRedirectionPipe(PipelineProcessor pipelineProcessor)
         {
-            if (pipelineProcessor == null) throw PSTraceSource.NewArgumentNullException(nameof(pipelineProcessor));
-            if (_redirectionPipes == null)
-                _redirectionPipes = new List<PipelineProcessor>();
+            if (pipelineProcessor is null)
+            {
+                throw PSTraceSource.NewArgumentNullException(nameof(pipelineProcessor));
+            }
```

Comment:
If we want being short:
```suggestion
            pipelineProcessor ?? throw PSTraceSource.NewArgumentNullException(nameof(pipelineProcessor));
```

Reply:
> `pipelineProcessor ?? throw PSTraceSource.NewArgumentNullException(nameof(pipelineProcessor));`

You cannot do this, an expression cannot be used as a statement.